swiftpkg/internal/swift_registry_package.bzl

```diff
@@ -0,0 +1,88 @@
+"""Implementation for `swift_registry_package`."""
+
+load("@bazel_skylib//lib:dicts.bzl", "dicts")
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "update_attrs")
+load(":pkg_ctxs.bzl", "pkg_ctxs")
+load(":registry_configs.bzl", "registry_configs")
+load(":repo_rules.bzl", "repo_rules")
+load(":repository_files.bzl", "repository_files")
+
+def _remove_bazel_files(repository_ctx, directory):
+    files = ["BUILD.bazel", "BUILD", "WORKSPACE", "WORKSPACE.bazel"]
+    for file in files:
+        repository_files.find_and_delete_files(repository_ctx, directory, file)
```

Comment:
Let's move this into `repository_files` and then update `swift_package` and this code to use it.

Reply:
done!